internal/manager/config.go

```diff
@@ -231,8 +224,16 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	// SIGTERM or SIGINT signal delay
 	flagSet.DurationVar(&c.TermDelay, "term-delay", time.Second*0, "The time delay to sleep before SIGTERM or SIGINT will shut down the Ingress Controller")
 
-	// Deprecated flags
+	// Konnect
+	flagSet.BoolVar(&c.Konnect.ConfigSynchronizationEnabled, "konnect-sync-enabled", false, "Enable synchronization of data plane configuration with a Konnect runtime group.")
+	flagSet.StringVar(&c.Konnect.RuntimeGroup, "konnect-runtime-group", "", "An ID of a runtime group (not name!) that is to be synchronized with data plane configuration.")
```

Comment:
Perhaps it's worth changing this to include `id`/`ID` in the name to avoid the hint in the flag description?